apps/web/src/components/v2Editor/customBlocks/python/PythonOutput.tsx

```diff
@@ -106,16 +107,33 @@ interface ItemProps {
   canFixWithAI: boolean
 }
 export function PythonOutput(props: ItemProps) {
+  const onExportToPNG = () => {
+    if (props.output.type !== 'image' || props.output.format !== 'png') return
+
+    downloadFile(
+      `data:image/${props.output.format};base64, ${props.output.data}`,
+      'Generated Python image'
```

Comment:
Isn't this name supposed to be the filename? I think we could use the block's ID followed by `.png` here

Reply:
Done.

Adding, we don't need to add `.png` in the file name due to the MIME type (`props.output.format`) defined in the file url.